code/modules/admin/verbs/adminsay.dm

```diff
@@ -21,15 +21,16 @@
 			SSredis.publish("byond.asay", json_encode(data))
 
 		for(var/client/C in GLOB.admins)
+			var/temp_message = msg
```

Comment:
Is making a new var necessary?

Reply:
Yes because it fixed a bug that currently exists.

When you ping an admin, it shows up red on their end, however some admins will see it on their end due to the message being overwritten, and some of them won't depending on where they are in the loop.